<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionairport</class>
 <widget class="QMainWindow" name="gestionairport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKYUP AIRLINE </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>167</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; color:#ffffff;&quot;&gt;Gestion Airport &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tab_gestionairport">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter un airport</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>108</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant Airport</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>108</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom Airport </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>108</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Ville </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idairport">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>521</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomairport">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>521</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="villeairport">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>521</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>300</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="textSetting">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>621</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer un airport </string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>108</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant Airport</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idairport_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>471</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_suppairport">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>641</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>330</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="testSupp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Consulter les airports</string>
     </attribute>
     <widget class="QTableView" name="tableView_gestionairport">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>641</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
